### DISCUSSION TITLE
Begginer in Python in search of basics exercises

### DISCUSSION THREAD

**Discussion by LaloCotanda** [codewars/codewars.com#2611] [Q&A]
Hello everybody.

I'm a begginer learning Python from 0. I need some basics exercises to practice and learn better. Anyone can help me? I'll appreciate a lot if someone help me. In Spanish or English, I don't care.

Thank you very much.

**Comment by faiz-ali-khan-developer** [comment]
if you are a beginner learning python from scratch then I would suggest you follow some books, which I think will be great for you. Following books will surely help and they are also easily available on the internet in PDF format.
- Python crash course.
- Automate the boring stuff with python.
- Cracking codes with python
- Invent with python.

**Reply by iorchalvarez** [reply]
If you don't like reading that much then you can also watch some videos and follow what they're doing in there. I would recommend to stop after each exercise and analyze what they did to understand. Plus also modify a few things and see what happens with the changes.

**Reply by LaloCotanda** [reply]
Thank you very much for your advices.

**Comment by mustafa975** [comment]
Iam begginer in python please help me and give me a write path that i learned python

**Reply by formerglory87** [reply]
If you have the extra money. codecademy has a great program that will teach you python. They have the step by step instructions telling how it should be done. At the end of each section there is a project that tells you what needs done while you "free code" what you just learned. Outside of that, there are plenty of books and youtube videos. ( I personally like "Programming with Mosh")
